### PR TITLE
partially switching to nx

### PR DESCRIPTION
**Pull request by gabrielbadarau**

### CODE REVIEW

File: apps/Project1/karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+module.exports = function (config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    plugins: [
+      require('karma-jasmine'),
+      require('karma-chrome-launcher'),
```
Comment (andragugu) [inline apps/Project1/karma.conf.js:10]: but if you're using Jest, you don't need jasmine anymore

File: apps/Project1/src/app/users/users.service.ts [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
+import { Iusers } from '../../../../../libs/core-data/src';
+
 
 @Injectable()
 export class UsersService extends EntityCollectionServiceBase<Iusers> {
   private toastMessageSubject = new Subject<[value: boolean, action: string]>();
   toastMessageAction$ = this.toastMessageSubject.asObservable();
 
-  private choiceToDeleteUserSubject = new Subject<Iusers>();
-  choiceToDeleteUserAction$ = this.choiceToDeleteUserSubject.asObservable();
```
Comment (andragugu) [inline apps/Project1/src/app/users/users.service.ts:12]: here, you don't need to reassign, you can chain to be observable by default

File: apps/Project1/karma.conf.js (hunk 1 above)
Comment (andragugu) [reply apps/Project1/karma.conf.js:10]: so karma conf can be deleted

Comment (gabrielbadarau) [reply apps/Project1/karma.conf.js]: test

**Pull request merged**